canonical-web-and-design/maas-ui
# refactor(ui): refactor available storage table to work with controllers

## Done

- Refactored machine available storage table to work with controllers

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the React storage tab of a controller and check that the data matches what you see in the AngularJS version
- Check that you can still edit the storage of a Ready or Allocated machine

## Fixes

Fixes canonical-web-and-design/app-tribe#999

## Screenshot

![Screenshot 2022-06-01 at 14-29-13 bolla storage bolla MAAS](https://user-images.githubusercontent.com/25733845/171328301-9b6cf942-32c4-4cbb-b090-e04214c23f43.png)
